tests/unittest_scoped_nodes.py

```diff
@@ -956,7 +965,12 @@ def test_cls_special_attributes_1(self) -> None:
             self.assertEqual(
                 len(cls.getattr("__doc__")), 1, (cls, cls.getattr("__doc__"))
             )
-            self.assertEqual(cls.getattr("__doc__")[0].value, cls.doc)
+            with pytest.warns(DeprecationWarning) as records:
+                self.assertEqual(cls.getattr("__doc__")[0].value, cls.doc)
+                # TODO: This should be 1 once we update the use of 'doc' internally
+                assert len(records) == 2
+            # TODO: The doc_node attribute needs to be set in the raw_building process
```

Comment:
I'm not sure if this should be in here, but I think we should test this and with this TODO we will stumble upon this when we change L969-971.